pkg/snapshot/generate_snapshot_name.go

```diff
@@ -9,16 +9,10 @@ import (
 	"encoding/base32"
 	"fmt"
 	"strings"
-
-	"github.com/open-component-model/ocm-controller/api/v1alpha1"
 )
 
-// GetSnapshotNameForObject returns an existing snapshot name or generates a random one in the case
-// that the name does not exist
-func GetSnapshotNameForObject(name string, obj v1alpha1.SnapshotProducer) (string, error) {
-	if obj.GetSnapshotName() != "" {
-		return obj.GetSnapshotName(), nil
-	}
+// GetSnapshotName generates a random snapshot name.
+func GetSnapshotName(name string) (string, error) {
```

Comment:
should be `GenerateSnapshotName` now that it doesn't return an existing snapshot name.

Reply:
Good call.